libs/datasets/manual_filter.py

```diff
@@ -240,14 +240,46 @@ def _transform_one_override(
     override: Mapping, cbsa_to_counties_map: Mapping[Region, List[Region]]
 ) -> Filter:
     region_str = override["region"]
+    include_str = override["include"]
+
+    if not isinstance(region_str, str):
+        raise ValueError(f"Invalid 'region': {region_str}")
+    if not isinstance(include_str, str):
+        raise ValueError(f"Invalid 'include': {include_str}")
+
+    # Split on commas.
+    region_strs = [single_region_str.strip() for single_region_str in region_str.split(",")]
+    # Convert each region string to 1+ regions and aggregate.
+    regions_included = []
+    for region_str in region_strs:
+        for region in _transform_region_str(region_str, include_str, cbsa_to_counties_map):
+            regions_included.append(region)
```

Comment:
FWIW- I tried to do this all with a list comprehension but to get the regions list properly flattened it seemed easier to just do it procedurally like this... 🤷

Reply:
I think `regions_included.extend(transform_region_str(region_str, include_str, cbsa_to_counties_map))` eliminates one loop without much complication.